plugins/search/src/components/SidebarSearch/SidebarSearch.tsx

```diff
@@ -19,9 +19,13 @@ import { useNavigate } from 'react-router-dom';
 import { rootRouteRef } from '../../plugin';
 
 import { SidebarSearchField } from '@backstage/core-components';
-import { useRouteRef } from '@backstage/core-plugin-api';
+import { useRouteRef, IconComponent } from '@backstage/core-plugin-api';
 
-export const SidebarSearch = () => {
+type SidebarSearchProps = {
+  icon?: IconComponent;
+};
+
+export const SidebarSearch = (props: SidebarSearchProps) => {
```

Comment:
Do you think it would be valuable providing this prop to the SearchBarModal component as well? https://github.com/backstage/backstage/blob/afa656ae6b7139df31086eac704ebd3445443836/plugins/search/src/components/SidebarSearchModal/SidebarSearchModal.tsx#L27

That is used if one chose to use the search modal https://backstage.io/docs/features/search/getting-started#using-the-search-modal

Reply:
Yes, you are right. I will update it.

Reply:
Yes, you are right. I will update it.

Reply:
Updated, please review it again :)

Reply:
I will mark this as resolve if you don't mind :)